src/pages/organisation-unit-distribution-report/connectOrganisationUnitDistributionReport.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux'
+import {
+    loadingChartImageUrlStart,
+    loadingChartImageUrlSuccessWithFeedback,
+    loadingChartImageUrlErrorWithFeedback,
+} from '../../redux/actions/orgUnitDistReport'
+import { loadGroupSetOptions } from '../../redux/actions/orgUnitGroupSets'
+import { loadReport } from '../../redux/actions/orgUnitDistReport'
+import { getChartImageUrl } from '../../redux/selectors/orgUnitDistReport/getChartImageUrl'
+import { getLoading } from '../../redux/selectors/orgUnitDistReport/getLoading'
+import { getIsActionEnabled } from '../../redux/selectors/orgUnitDistReport/getIsActionEnabled'
+import getTranformedTableData from '../../redux/selectors/reportingRateSummary/getTranformedTableData.js'
```

Comment:
Ultimately, we need a different selector here. But I guess we can leave this out of scope for the current branch.